packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -494,14 +485,10 @@ export class IndexedDbPersistence implements Persistence {
 
   runTransaction<T>(
     action: string,
-<<<<<<< HEAD
     requirePrimaryLease: boolean,
     transactionOperation: (
-      transaction: PersistenceTransaction
+      transaction: IndexedDbTransaction
```

Comment:
This should be `PersistenceTransaction`. The same mistake exists in master, which I will fix.

Reply:
I will backport your fix to master in the next merge.